### PR TITLE
Add plan to test cloud providers

### PR DESCRIPTION
**Pull request by benarent**
Feedback from CS team that we didn't test all Cloud Setups. Adding a manual check.

### CODE REVIEW

File: docs/testplan.md [hunk 1]
```diff
@@ -431,3 +431,25 @@ Observe prometheus metrics for goroutines, open files, RAM, CPU, Timers and make
...
+### AWS 
+
+- [ ] Deploy Teleport to AWS. Using DynamoDB & S3
+- [ ] Deploy Teleport Entperise to AWS. Using HA Setup https://gravitational.com/teleport/docs/aws_terraform_guide/
+
+### GCP 
+
+- [ ] Deploy Teleport to GCP. Using Cloud Firestore & Cloud Storage
+- [ ] Deploy Teleport to GKE. Google Kubernetes engine. 
+- [ ] Deploy Teleport Entperise to GCP.
```
Comment (webvictim) [inline docs/testplan.md:447]: Enterprise

File: docs/testplan.md [hunk 2]
```diff
@@ -431,3 +431,25 @@ Observe prometheus metrics for goroutines, open files, RAM, CPU, Timers and make
...
+
+### IBM
+
+- [ ] Deploy Teleport to IBM Cloud. Using IBM Database for etcd & IBM Object Store
+- [ ] Deploy Teleport to IBM Cloud Kubernetes. 
+- [ ] Deploy Teleport Entperise to IBM Cloud.
```
Comment (webvictim) [inline docs/testplan.md:453]: Enterprise

File: docs/testplan.md [hunk 3]
```diff
@@ -431,3 +431,25 @@ Observe prometheus metrics for goroutines, open files, RAM, CPU, Timers and make
...
 Load system with tsh bench to the capacity and publish maximum numbers of concurrent sessions with interactive
 and non interactive tsh bench loads.
+
+
+## Teleport with Cloud Providers
+
+### AWS 
+
+- [ ] Deploy Teleport to AWS. Using DynamoDB & S3
+- [ ] Deploy Teleport Entperise to AWS. Using HA Setup https://gravitational.com/teleport/docs/aws_terraform_guide/
```
Comment (webvictim) [inline docs/testplan.md:441]: Enterprise

**Comment by webvictim** [comment]
```
docs/testplan.md:### AWS
docs/testplan.md:### GCP
docs/testplan.md:- [ ] Deploy Teleport to GKE. Google Kubernetes engine.
docs/testplan.md:- [ ] Deploy Teleport to IBM Cloud Kubernetes.
trailing whitespace found in docs/ (see above)
run 'make docs-fix-whitespace' to fix it
make: *** [Makefile:215: docs-test-whitespace] Error 1
```

**Review by travelton — APPROVED** [review]
w/ Gus' fixes.

File: docs/testplan.md (hunk 3 above)
Comment (benarent) [reply docs/testplan.md:441]: needed more coffee today :-P

**Review by webvictim — APPROVED** [review]

**Pull request merged**